larger_than_memory.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<p>Make bigger at the end, this is now somewhat scalable.</p><p> </p><p>I am looking into synthea dataset here <a href="https://synthea.mitre.org/downloads" target="_blank">https://synthea.mitre.org/downloads</a>, time-intense to make this meaningful though and especially s.t. people can actually use this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yeah it should be bigger. You might need to explain what the chunks are?</p>